symfony/plugins/orangehrmMarketPlacePlugin/lib/dao/MarketplaceDao.php

```diff
@@ -22,6 +22,9 @@
  */
 class MarketplaceDao
 {
+    const INSTALLED = 'Installed';
```

Comment:
better to rename as ADDON_STATUS_ prefix .. 

ADDON_STATUS_INSTALLED
ADDON_STATUS_REQUESTED
ADDON_STATUS_PAID